### ISSUE TITLE
Defualt epoch&lr

### ISSUE THREAD

**Issue by hatmore** [Yiming-M/CLIP-EBC#5]
Why are the default settings for epoch and learning rate set to 2600 and 1.00e-06, respectively? Wouldn’t this make the training process very slow?

**Comment by Yiming-M** [comment]
Hi @hatmore ,

We are using a warmup training schedule in which the learning rate will increase from `warmup_lr` (default to `1e-6`) to `lr` (default to `1e-4`) during the first `warmup_epochs` (default to `50`) to stabilise the training process. We haven't done any experiments on what is the best learning rate setting, so perhaps the strategy that we are using is suboptimal. Feel free to explore other hyper-parameter settings, and please do let us know that there are better recipes.

Many thanks,
Yiming
